src/backend/drm/mod.rs

```diff
@@ -113,6 +113,16 @@ fn warn_legacy_fb_export() {
     });
 }
 
+/// Hint for DRM backend on how the surface should be presented
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PresentationMode {
+    /// Vertical synchronization
+    VSync,
+    /// Thearing presentationres
+    ASync,
```

Comment:
```suggestion
    Async,
```
It's one word after all, also line above Tearing :see_no_evil:

Reply:
Done